packages/web-frontend/src/containers/MapTableModal/MapTableModal.js

```diff
@@ -66,24 +66,25 @@ MapTableModalComponent.propTypes = {
     }),
   ),
   currentCountry: PropTypes.object,
-  currentMeasure: PropTypes.object,
+  currentMapOverlay: PropTypes.object,
 };
 
 MapTableModalComponent.defaultProps = {
   measureOptions: null,
   measureData: null,
   currentCountry: null,
-  currentMeasure: null,
+  currentMapOverlay: null,
 };
 
 const mapStateToProps = state => {
   const { measureOptions } = state.map.measureInfo;
-  const currentMeasure = selectCurrentMeasure(state);
+  // TODO: select multiple map overlays
```

Comment:
@billli0 tip (no need to change this): you can include the card number in the TODO for easier tracking, eg

```suggestion
  // TODO: select multiple map overlays - PHX-1
```

Reply:
Good idea! It helps to find in the future.